CLA.txt

```diff
@@ -109,3 +109,5 @@ and interest in and to Your Contributions.
 8. You agree to notify Unlock of any facts or circumstances of
    which you become aware that would make these representations
    inaccurate in any respect.
+
+PauliB14
```

Comment:
ha no that's in this file: https://github.com/unlock-protocol/unlock/blob/master/.clabot